src/exiv2.cpp

```diff
@@ -40,7 +40,7 @@ int main(int argc, char* const argv[])
 
 #ifdef EXV_ENABLE_NLS
     setlocale(LC_ALL, "");
-    const std::string localeDir = Exiv2::getProcessPath() + EXV_LOCALEDIR;
+    const std::string localeDir = EXV_LOCALEDIR[0] == '/' ? EXV_LOCALEDIR : Exiv2::getProcessPath() + "/" + EXV_LOCALEDIR;
```

Comment:
It would be nicer to be able to use `std::filesystem::path(Exiv2::getProcessPath()) /  EXV_LOCALEDIR` but that requires C++17:

https://en.cppreference.com/w/cpp/filesystem/path

Reply:
Only C++11 is allowed currently.

Reply:
@tbeu is correct.  Currently Exiv2 v0.27-maintenance is written to the C++98 standard.  When it compiles with C++11 or C++14, you should set the flags: ` -DCMAKE_CXX_STANDARD=11 -DCMAKE_CXX_FLAGS=-Wno-deprecated`

The 'master' branch builds with C++11.  When we decide to "modernise" the C++ code, KDE requested that we avoid dependency on C++14 and later.

Exiv2 has always avoided the Boost.  We want to be able to build Exiv2 with the platform standard C++ compiler and minimal dependencies.  To support XMP, we require Expat.  To support PNG, we require zlib.  Both XMP and PNG are optional.  All other dependencies (localisation, charset conversions and gtest) are optional.

Reply:
Yeah, I suspected something like that. Fortunately, with conditional expressions it is not that much longer.